notebooks/video_clips/VideoClips.ipynb

```diff
@@ -0,0 +1,963 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "2ef71144-e2c1-4696-a82a-a9c49dae001e",
+   "metadata": {},
+   "source": [
+    "# Creating Video Clips\n",
+    "This notebook demonstrates using a YOLOv4 model to process individual frames and use the resulting output to generate information about the video.\n",
+    "We use detections of labels over sequential frames to generate Clips which describe the existance of those objects within a specific portion of the video."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8835379a-1c02-44d3-930c-f321279ee0bb",
+   "metadata": {},
+   "source": [
+    "# Install ApertureDB\n",
```

Comment:
```suggestion
    "## Install ApertureDB\n",
```

Reply:
I like the structuring in this example: https://docs.aperturedata.io/HowToGuides/Applications/similarity_search
Am looking where we start defining standards but lost @bovlb  PRs

Reply:
But we do need to point to cloud setup in all these.